Makefile

```diff
@@ -285,18 +289,69 @@ ifdef MACOS_CODESIGN_IDENT
 	scripts/sign-macos-pkg.sh $(STATUS_CLIENT_DMG) $(MACOS_CODESIGN_IDENT)
 endif
 
-STATUS_CLIENT_EXE ?= pkg/Status.exe
-
-# not implemented yet
-# $(STATUS_CLIENT_EXE): nim_status_client
+NIM_WINDOWS_PREBUILT_DLLS ?= tmp/windows/tools/pcre.dll
+
+$(NIM_WINDOWS_PREBUILT_DLLS):
+	echo -e "\e[92mFetching:\e[39m prebuilt DLLs from nim-lang.org"
+	rm -rf tmp/windows
+	mkdir -p tmp/windows/tools
+	cd tmp/windows/tools && \
+	wget https://nim-lang.org/download/dlls.zip && \
+	unzip dlls.zip
+
+nim_windows_launcher: | deps
+	$(ENV_SCRIPT) nim c -d:debug --outdir:./bin --passL:"-static-libgcc -Wl,-Bstatic,--whole-archive -lwinpthread -Wl,--no-whole-archive" src/nim_windows_launcher.nim
+
+STATUS_CLIENT_ZIP ?= pkg/Status.zip
+
+$(STATUS_CLIENT_ZIP): nim_status_client nim_windows_launcher $(NIM_WINDOWS_PREBUILT_DLLS)
+	rm -rf pkg/*.zip
+	rm -rf tmp/windows/dist
+	mkdir -p tmp/windows/dist/Status/bin
+	mkdir -p tmp/windows/dist/Status/resources
+	mkdir -p tmp/windows/dist/Status/vendor
+	cp windows-install.txt tmp/windows/dist/Status/INSTALL.txt
+	unix2dos -k tmp/windows/dist/Status/INSTALL.txt
+	# cp LICENSE tmp/windows/dist/Status/LICENSE.txt
+	# unix2dos -k tmp/windows/dist/Status/LICENSE.txt
```

Comment:
Are these two lines necessary?

Reply:
Once we have a LICENSE file in the root of the repo, my plan is to uncomment those lines.